charter.html

```diff
@@ -156,8 +156,8 @@ <h2>Scope</h2>
         <p>
             The Working Group will specify new web platform features intended to
```

Comment:
The Working Group will specify new web platform features (new web platform APIs) intended to